File: benchmarks/baselines/cross-language/fib.go
package main

import "os"

func fib(n int64) int64 {
	if n < 2 { return n }
	return fib(n-1) + fib(n-2)
}

func main() {
	if fib(40) == 102334155 { os.Exit(0) } else { os.Exit(1) }
}

File: benchmarks/baselines/cross-language/fib.rs
fn fib(n: i64) -> i64 {
    if n < 2 { n } else { fib(n - 1) + fib(n - 2) }
}

fn main() {
    let result = fib(40);
    std::process::exit(if result == 102334155 { 0 } else { 1 });
}
